src/components/Card/index.tsx

```diff
@@ -23,8 +25,7 @@ const Card = ({
         <Icon className="card-icon" />
         <OverflowMenu className="card-overflow" ariaLabel="overflow-menu">
           <OverflowMenuItem tabIndex="0" itemText={highlighted ? 'Dehighlight shortcut' : 'Highlight shortcut'} onClick={highlightFunction} />
-          <OverflowMenuItem itemText="Item 2" />
-          <OverflowMenuItem itemText="Item 3" />
+          <OverflowMenuItem itemText="Delete shortcut" onClick={deleteFunction} />
```

Comment:
Change the text here, "Remove Favorite" reflects better the function and it is more user friendly :)

Reply:
The text was based on figma/jira, it's a change we can suggest to the UX team